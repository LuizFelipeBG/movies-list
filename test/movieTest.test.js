import React from 'react';
import { render, fireEvent } from '@testing-library/react-native';
import FilterMovie from '../src/container/filterMovie';

describe("Filter Movie tests", () => {

  test('examples of some things', async () => {
    const { getByTestId, getByText, queryByTestId, toJSON } = render(<FilterMovie />)
    const famousProgrammerInHistory = 'Ada Lovelace'
  
    const inputName = getByTestId('InputFilterName')
    fireEvent.changeText(inputName, famousProgrammerInHistory)

    const inputYear = getByTestId('InputFilterYear')
    fireEvent.changeText(inputYear, famousProgrammerInHistory)    
  
    const button = getByText('filterSubmit')
    fireEvent.press(button)
  })
})