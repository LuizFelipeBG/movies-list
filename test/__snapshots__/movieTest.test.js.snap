// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Fliter Movie tests renders correctly component for filter movie 1`] = `
<View
  style={Object {}}
>
  <Text>
    Filter
  </Text>
</View>
`;

exports[`renders correctly 1`] = `
<View
  style={Object {}}
>
  <Text>
    Filter
  </Text>
</View>
`;
